streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/core/resource/Container.java

```diff
@@ -0,0 +1,86 @@
+package org.ray.streaming.runtime.core.resource;
+
+import com.google.common.base.MoreObjects;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.ray.api.id.UniqueId;
+
+/**
+ * Resource manager unit abstraction.
```

Comment:
It’s not clear to me what `Container ` means? It's a ray node or ray worker or a external container such as k8s pod?

Reply:
Container is the abstraction of streaming internal resources, which corresponds to the container of yarn and the pod of k8s.

Reply:
How does `ResourceManager` get external resource from `yarn/k8s`? What will it do after got resource? When and who create `raylet/store` process? Can you add more comments here or other places if more appropriate?

Reply:
```ResourceManager``` gets the resource according to the raylet process. Yarn or k8s will pull up a raylet process in the container/pod.

Reply:
> which corresponds to the container of yarn and the pod of k8s

Ray uses the term "node". How about also using "node" here?

Reply:
Streaming doesn't whether the underlying deployment is in yarn、k8s or direct ray cluster nodes, so it abstracts a container concept. So we think the name ```container``` is more suitable.